include/beman/iterator_interface/CMakeLists.txt

```diff
@@ -9,9 +9,12 @@ target_sources(
          beman_iterator_interface_headers
          TYPE
          HEADERS
+         BASE_DIRS
+         ${PROJECT_BINARY_DIR}/include
+         ${PROJECT_SOURCE_DIR}/include
          FILES
-         iterator_interface.hpp
-         iterator_interface_access.hpp
-         detail/stl_interfaces/config.hpp
-         detail/stl_interfaces/fwd.hpp
-         detail/stl_interfaces/iterator_interface.hpp)
+         ${CMAKE_CURRENT_SOURCE_DIR}/iterator_interface.hpp
```

Comment:
CMAKE_CURRENT_SOURCE_DIR is redundant for a FILE_SET. It's the root for the source for files in the set.

Reply:
OK